test/unit_tests.html

```diff
@@ -2405,7 +2405,66 @@ <h4>Test HTML</h4>
         assert.equal($("#test-cancel-with-ic-on-success").text(), "");
       });
   </script>
-
+	
+  <div id="test-ic-push-url-and-ic-use-transition">
+    <a id="page1-tab"
+      href=""
+      ic-get-from="/page1"
+      ic-target="#test-ic-push-url-and-ic-use-transition-content"
+      ic-push-url="true"
+      ic-indicator="#ic-push-url-and-ic-use-transition-indicator">
+      Page1
+    </a>
+    <a id="page2-tab"
+      href=""
+      ic-get-from="/page2"
+      ic-target="#test-ic-push-url-and-ic-use-transition-content"
+      ic-push-url="true"
+      ic-indicator="#ic-push-url-and-ic-use-transition-indicator">
+      Page2
+    </a>
+    <i id="ic-push-url-and-ic-use-transition-indicator" class="fa fa-spinner fa-spin ic-use-transition"></i>
+    <div id="test-ic-push-url-and-ic-use-transition-content" ic-history-elt="">
+      <h1>Content</h1>
+    </div>
+  </div>
+  <script>
+    QUnit.test("ic-use-transition should work when using ic-push-url", function (assert) {
+      // don't run test locally
+      if (window.location.hostname == "") {
```

Comment:
This is actually a question for @chg20: I see that this `if` block is copied from the existing test for "Basic history support works", where you added it sometime after originally writing that test. What problem does it prevent? Is it a problem in all browsers?

Reply:
I tried this myself, and I see now that it does indeed fail when run via `file:///...` without this guard, but it passes when run via `http://localhost/...` (in Chrome).